hudi-utilities/src/main/java/org/apache/hudi/utilities/config/HoodieDeltaStreamerConfig.java

```diff
@@ -0,0 +1,283 @@
+
+  public static final ConfigProperty<String> TABLES_TO_BE_INGESTED_PROP = ConfigProperty
+      .key(INGESTION_PREFIX + "tablesToBeIngested")
+      .noDefaultValue()
+      .withDocumentation("Comma separated names of tables to be ingested in the format <database>.<table>, for example db1.table1,db1.table2");
+
+  public static final ConfigProperty<String> TARGET_BASE_PATH_PROP = ConfigProperty
+      .key(INGESTION_PREFIX + "targetBasePath")
+      .noDefaultValue()
+      .withDocumentation("The path to which a particular table is ingested. The config is specific to HoodieMultiTableDeltaStreamer"
+          + " and overrides path determined using option `--base-path-prefix` for a table");
+
+  /**
+   * Delta Streamer Schema Provider related config.
+   */
+  @Immutable
+  @ConfigClassProperty(name = "DeltaStreamer Schema Provider Configs",
+      groupName = ConfigGroups.Names.DELTA_STREAMER,
+      description = "Configurations related to source and target schema for DeltaStreamer.")
+  public static class HoodieDeltaStreamerSchemaProviderConfig extends HoodieConfig {
+
+    private static final String DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX = DELTA_STREAMER_CONFIG_PREFIX + "schemaprovider.";
+
+    public static final ConfigProperty<String> SCHEMA_POST_PROCESSOR_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "schema_post_processor")
+        .noDefaultValue()
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> SRC_SCHEMA_REGISTRY_URL_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "registry.url")
+        .noDefaultValue()
+        .withDocumentation("The schema of the source you are reading from e.g. https://foo:bar@schemaregistry.org");
+
+    public static final ConfigProperty<String> TARGET_SCHEMA_REGISTRY_URL_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "registry.targetUrl")
+        .noDefaultValue()
+        .withDocumentation("The schema of the target you are writing to e.g. https://foo:bar@schemaregistry.org");
+
+    public static final ConfigProperty<String> SCHEMA_CONVERTER_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "registry.schemaconverter")
+        .noDefaultValue()
+        .withDocumentation("");
+
+    public static final ConfigProperty<Boolean> SPARK_AVRO_POST_PROCESSOR_PROP_ENABLE = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "spark_avro_post_processor.enable")
+        .defaultValue(false)
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> DELETE_COLUMN_POST_PROCESSOR_COLUMN_PROP = ConfigProperty
+        .key(DELTA_STREAMER_SCHEMAPROVIDER_CONFIG_PREFIX + "schema_post_processor.delete.columns")
+        .noDefaultValue()
+        .withDocumentation("");
+
+    public static final ConfigProperty<String> SCHEMA_POST_PROCESSOR_ADD_COLUMN_NAME_PROP = ConfigProperty
```

Comment:
I see these are moved from `BaseSchemaPostProcessorConfig`.  Should schema post processor configs still be kept in a separate class, extending `HoodieConfig`?